src/main/kotlin/com/deviceinsight/helm/AbstractHelmMojo.kt

```diff
@@ -39,24 +40,32 @@ abstract class AbstractHelmMojo : AbstractMojo() {
 	private var chartName: String? = null
 
 	protected fun executeCmd(
-		vararg cmd: String,
+		cmd: List<String>,
 		directory: File = target(),
 		redirectOutput: ProcessBuilder.Redirect = ProcessBuilder.Redirect.PIPE
 	) {
-		val proc = ProcessBuilder(cmd.toList())
+		val proc = ProcessBuilder(cmd)
 			.directory(directory)
 			.redirectOutput(redirectOutput)
 			.redirectError(ProcessBuilder.Redirect.PIPE)
 			.start()
 
+		val stdoutPrinter = thread(name = "Stdout printer") {
+			proc.inputStream.bufferedReader().lines().forEach { log.debug("Output: $it") }
+		}
+
+		val stderrPrinter = thread(name = "Stderr printer") {
+			proc.errorStream.bufferedReader().lines().forEach { log.error("Output: $it") }
```

Comment:
I hope you don't mind me spawning two threads only for printing out the stdout/stderr stream